KarhooUISDK/CommonUI/MapMVP/AppleMapKit/KarhooMKMapView.swift

```diff
@@ -247,22 +264,25 @@ extension KarhooMKMapView: MKMapViewDelegate {
     }
 
     func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
-        let view = MKAnnotationView(annotation: annotation, reuseIdentifier: "pin")
-        let customAnnotation = annotation as? MapAnnotationViewModel
-        view.image = customAnnotation?.backgroundIcon
-        
-        if let iconImage = customAnnotation?.foregroundIcon {
-            let icon = UIImageView(image: iconImage)
-            icon.translatesAutoresizingMaskIntoConstraints = false
-            icon.contentMode = .scaleAspectFit
-            view.addSubview(icon)
-            
-            icon.centerX(inView: view)
-            icon.centerY(inView: view, constant: -6)
-            icon.anchor(width: 16, height: 16)
+        if annotation is MKUserLocation {
+            return nil
```

Comment:
In what scenario does it pass through this line? I can't seem to find a case.

Reply:
Default MKMapView user location marker is `MKUserLocation`.